Specs/Sequence/English/PhoneNumberModel.json

```diff
@@ -3343,10 +3340,10 @@
     "NotSupportedByDesign": "python",
     "Results": [
       {
-        "Text": "19 35831647",
```

Comment:
This one I think is a little ambiguous. I'm fine if we extract the number, but I don't think the tagged text should include the mismatched '('.